2024/03/16 11:31:25 [error] 1#0: init_by_lua error: /usr/local/share/lua/5.1/kong/init.lua:656: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: FATAL: the database system is starting up
stack traceback:
	[C]: in function 'assert'
	/usr/local/share/lua/5.1/kong/init.lua:656: in function 'init'
	init_by_lua(nginx-kong.conf:41):3: in main chunk
2024/03/16 11:31:27 [error] 1#0: init_by_lua error: /usr/local/share/lua/5.1/kong/init.lua:656: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: FATAL: the database system is starting up
stack traceback:
	[C]: in function 'assert'
	/usr/local/share/lua/5.1/kong/init.lua:656: in function 'init'
	init_by_lua(nginx-kong.conf:41):3: in main chunk
2024/03/16 11:31:30 [error] 1#0: init_by_lua error: /usr/local/share/lua/5.1/kong/init.lua:656: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: FATAL: the database system is starting up
stack traceback:
	[C]: in function 'assert'
	/usr/local/share/lua/5.1/kong/init.lua:656: in function 'init'
	init_by_lua(nginx-kong.conf:41):3: in main chunk
2024/03/16 11:31:33 [error] 1#0: init_by_lua error: /usr/local/share/lua/5.1/kong/init.lua:656: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: FATAL: the database system is starting up
stack traceback:
	[C]: in function 'assert'
	/usr/local/share/lua/5.1/kong/init.lua:656: in function 'init'
	init_by_lua(nginx-kong.conf:41):3: in main chunk
2024/03/16 11:31:36 [notice] 1#0: [lua] init.lua:776: init(): [request-debug] token for request debugging: 06cd2656-660f-4b76-b739-0c2fd4b2af6c
2024/03/16 11:31:36 [notice] 1#0: using the "epoll" event method
2024/03/16 11:31:36 [notice] 1#0: openresty/1.25.3.1
2024/03/16 11:31:36 [notice] 1#0: OS: Linux 5.15.133.1-microsoft-standard-WSL2
2024/03/16 11:31:36 [notice] 1#0: getrlimit(RLIMIT_NOFILE): 1048576:1048576
2024/03/16 11:31:36 [notice] 1#0: start worker processes
2024/03/16 11:31:36 [notice] 1#0: start worker process 1279
2024/03/16 11:31:36 [notice] 1#0: start worker process 1280
2024/03/16 11:31:36 [notice] 1#0: start worker process 1281
2024/03/16 11:31:36 [notice] 1#0: start worker process 1282
2024/03/16 11:31:36 [notice] 1#0: start worker process 1283
2024/03/16 11:31:36 [notice] 1#0: start worker process 1284
2024/03/16 11:31:36 [notice] 1#0: start worker process 1285
2024/03/16 11:31:36 [notice] 1#0: start worker process 1286
2024/03/16 11:31:37 [notice] 1279#0: *1 [lua] warmup.lua:113: single_dao(): Preloading 'services' into the core_cache..., context: init_worker_by_lua*
2024/03/16 11:31:37 [notice] 1279#0: *1 [lua] warmup.lua:162: single_dao(): finished preloading 'services' into the core_cache (in 0ms), context: init_worker_by_lua*
2024/03/16 11:32:36 [error] 1279#0: *10289 [kong] handler.lua:56 [file-log] failed to open the file: Permission denied while logging request, client: 192.168.16.1, server: kong, request: "GET /api/hello HTTP/1.1", upstream: "http://172.18.176.1:8080/api/hello", host: "localhost:8000", request_id: "564f5b731cfabf34692e623bf2fe5c09"
